pkg/resource/v1/certconfig/resource.go

```diff
@@ -137,6 +137,19 @@ func isCertConfigModified(a, b *v1alpha1.CertConfig) bool {
 	return aVersion != bVersion
 }
 
+func toCertConfig(v interface{}) (*v1alpha1.CertConfig, error) {
+	if v == nil {
+		return nil, nil
+	}
+
+	certConfig, ok := v.(*v1alpha1.CertConfig)
+	if !ok {
+		return nil, microerror.Maskf(wrongTypeError, "expected '%T', got '%T'", certConfig, v)
+	}
+
+	return certConfig, nil
+}
```

Comment:
I think it's dead code.

Reply:
Actually it's used in `verifyCertConfigCreatedReactor()`. My thinking here was that it fits better right next to `toCertConfigs()`. Do you think it should be moved to `common_test.go` where it's used or even be inlined?

Reply:
I think for test it's ok to be inlined.

Reply:
Inlined.